generators/frontend/templates/angular-demo/src/app/app.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import { NavigationEnd, Router } from '@angular/router';
+import { environment } from '../environments/environment';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements OnInit {
+  public title = environment.siteName;
+
+  constructor(private router: Router) {}
+
+  ngOnInit() {
+    window.document.title = this.title;
+    this.router.events.subscribe(evt => {
+      if (!(evt instanceof NavigationEnd)) {
+        return;
+      }
+      window.scrollTo(0, 0);
```

Comment:
This whole block can be removed and replaced by `RouterModule.forRoot(routes, {scrollPositionRestoration: 'enabled'})` in the app module.